docs/get_start.rst

```diff
@@ -169,3 +169,30 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
         Network Adapter: Bridged Adapter
         Promiscuous Mode: on
 
+Relationship table of Infrasim command and standard server command
+--------------------------------------------------------------------
+
+Here we list a table to reflect the relationship of InfraSIM command and standard server command.
+
+    +---------------------------------------------+-----------------------------+
+    |standard server command                      |InfraSIM command             |
+    +=============================================+=============================+
+    |power on                                     |infrasim node start          |
+    +---------------------------------------------+-----------------------------+
+    |power off                                    |infrasim node stop           |
+    +---------------------------------------------+-----------------------------+
+    |removal server                               |infrasim node destroy        |
+    +---------------------------------------------+-----------------------------+
+    |reset                                        |infrasim node restart        |
+    +---------------------------------------------+-----------------------------+
+    |KVM - virtual keyboard, visual monitor       |infrasim VNC                 |
+    +---------------------------------------------+-----------------------------+
+    |configuration update                         |1. update yaml               |
+    |(node type, nic, processor, drive, memory)   |2. infrasim node stop        |
+    |                                             |3. infrasim node destroy     |
+    |                                             |4. infrasim node start       |
+    +---------------------------------------------+-----------------------------+
+    |add new server node                          |1. prepare yml               |
```

Comment:
"add new server node b-node"
1. compose b-node.yml
2. infrasim node config add --file=/path/to/b-node.yml
3. start
4. infrasim node config list
both a-node and b-node are there